kimchi/src/proof.rs

```diff
@@ -35,6 +36,9 @@ pub struct LookupEvaluations<Field> {
 }
 
 // TODO: this should really be vectors here, perhaps create another type for chunked evaluations?
+/// Polynomial evaluations contained in a `ProverProof`.
```

Comment:
Please add unrelated doc-comments in their own PR!

Reply:
🤦🏻 oh, I usually forget